src/Testing/src/xunit/SkipOnHelixAttribute.cs

```diff
@@ -42,6 +42,19 @@ public string SkipReason
                 return $"This test is skipped on helix";
             }
         }
+        
+        private bool ShouldSkip()
+        {
+            if (Queues == null) 
+            {
+                return true;
+            }
+            if (Queues.Contains("All.OSX")) 
+            {
+                return GetTargetHelixQueue().ToLowerInvariant().Contains("osx"));
+            }
+            return Queues.ToLowerInvariant().Split(';').Contains(GetTargetHelixQueue().ToLowerInvariant();
```

Comment:
```suggestion
            return Queues.ToLowerInvariant().Split(';').Contains(GetTargetHelixQueue().ToLowerInvariant());
```

nit: Suggest adding a variable set just above containing `GetTargetHelixQueue().ToLowerInvariant()`.